src/keypair.rs

```diff
@@ -9,8 +9,6 @@
 
 //! ed25519 keypairs.
```

Comment:
I renamed this whole module from `ed25519` -> `keypair` so I was able to re-export the entire `ed25519` crate as `ed25519_dalek::ed25519`.

It only contains the `Keypair` type anyway, so hopefully that makes sense.